src/main/java/com/scg/stop/auth/controller/AuthController.java

```diff
@@ -0,0 +1,86 @@
+package com.scg.stop.auth.controller;
+
+import com.scg.stop.auth.annotation.AuthUser;
+import com.scg.stop.auth.domain.UserToken;
+import com.scg.stop.auth.domain.request.LoginRequest;
+import com.scg.stop.auth.domain.request.RegisterRequest;
+import com.scg.stop.auth.domain.response.AccessTokenResponse;
+import com.scg.stop.auth.service.AuthService;
+import com.scg.stop.user.domain.AccessType;
+import com.scg.stop.user.domain.User;
+import com.scg.stop.user.domain.UserType;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseCookie;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CookieValue;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/auth")
+@Slf4j
+public class AuthController {
+    private static final int ONE_WEEK_SECONDS = 604800;
+
+    private final AuthService authService;
+
+    @PostMapping(value = "/login/kakao")
+    public ResponseEntity<AccessTokenResponse> kakaoLogin(
+            @RequestBody LoginRequest loginRequest,
+            HttpServletResponse response
+    ) {
+        UserToken userTokens = authService.login(loginRequest);
+
+        ResponseCookie cookie = ResponseCookie.from("refresh-token", userTokens.getRefreshToken())
+                .maxAge(ONE_WEEK_SECONDS)
+//                .secure(true)
+                .httpOnly(true)
+                .sameSite("None")
+                .domain(".stop.scg.skku.ac.kr")  // TODO: domain 수정
+                .path("/")
+                .build();
+
+        response.addHeader(HttpHeaders.SET_COOKIE, cookie.toString());
+        return ResponseEntity.ok(new AccessTokenResponse(userTokens.getAccessToken()));
+    }
+
+    //TODO: 추가 회원가입로직 작성
+    @PostMapping("/register")
+    public ResponseEntity<User> register(
+            @RequestBody RegisterRequest registerRequest,
```

Comment:
`@Valid`가 생략된 것 같습니다!

```suggestion
@RequestBody @Valid RegisterRequest registerRequest,
```

Reply:
8146820